src/lib/pages/backoffice/Document/DocumentDetails/DocumentDeleteModal/DocumentDeleteModal.js

```diff
@@ -73,11 +87,34 @@ export default class DocumentDeleteModal extends Component {
 
     const itemRefProps = {
       refType: 'Items',
-      onRefClick: this.handleOnItemRefClick,
+      onRefClick: this.handleOnEItemRefClick,
       getRefData: () =>
         itemApi.list(itemApi.query().withDocPid(documentPid).qs()),
     };
 
+    const borrowingRequestRefProps = {
+      refType: 'Borrowing requests',
+      onRefClick: this.handleOnBrwReqRefClick,
+      getRefData: () =>
+        borrowingRequestApi.list(
+          borrowingRequestApi.query().withDocPid(documentPid).qs()
+        ),
+    };
+
+    const ordersRefProps = {
+      refType: 'Orders',
+      onRefClick: this.handleOnOrderRefClick,
+      getRefData: () =>
+        orderApi.list(orderApi.query().withDocPid(documentPid).qs()),
+    };
+
+    const eitemRefProps = {
+      refType: 'E-items',
+      onRefClick: this.handleOnItemRefClick,
```

Comment:
Is the handle here correct? On Item and not on E-item? Same as above

Reply:
I changed the wrong one, thanks for staying alert !